libs/Matchmaking/test/Matchmaking.PeerDiscovery.Test/Matchmaking.PeerDiscovery.Test.csproj

```diff
@@ -12,8 +12,14 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
   </ItemGroup>
 
-  <ItemGroup>
+  <!-- Internal build, reference the projects directly -->
+  <ItemGroup Condition="True">
```

Comment:
> what condition should we use? An env variable? Existence of a file?
Or should we introduce new configs and build props? (Debug-Project, Debug-Nuget, Release-Project, Release-Nuget)

NuGet-(Debug|Release) configs might be the easiest way to test/change things locally (if it works straight-forwardly). Otherwise I think env var is fine. If we plan to only use this for CI testing then the simpler the better.